examples/inverse/plot_morph_stc_to_fsaverage_dipy.py

```diff
@@ -0,0 +1,231 @@
+
+References
+----------
+.. [1] Avants, B. B., Epstein, C. L., Grossman, M., & Gee, J. C. (2009).
+        Symmetric Diffeomorphic Image Registration with Cross- Correlation:
+        Evaluating Automated Labeling of Elderly and Neurodegenerative Brain,
+        12(1), 26-41.
+
+"""
+# Author: Tommy Clausner <tommy.clausner@gmail.com>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+import mne
+from mne.datasets import sample
+from mne.beamformer import make_lcmv, apply_lcmv
+
+from dipy.align.imwarp import SymmetricDiffeomorphicRegistration
+from dipy.align.metrics import CCMetric
+from dipy.align.transforms import (TranslationTransform3D,
+                                   RigidTransform3D,
+                                   AffineTransform3D)
+from dipy.align.imaffine import (transform_centers_of_mass,
+                                 MutualInformationMetric,
+                                 AffineRegistration)
+import nibabel as nib
+
+# uncomment the line(s) below if you encounter problems with ssl certification
+# import ssl
+# if hasattr(ssl, '_create_unverified_context'):
+#     ssl._create_default_https_context = ssl._create_unverified_context
+
+print(__doc__)
+
+##############################################################################
+# from examples/inverse/plot_lcmv_beamformer_volume.py
+
+# Setup paths
+data_path = sample.data_path()
+raw_fname = data_path + '/MEG/sample/sample_audvis_raw.fif'
+event_fname = data_path + '/MEG/sample/sample_audvis_raw-eve.fif'
+fname_fwd = data_path + '/MEG/sample/sample_audvis-meg-vol-7-fwd.fif'
+
+# Get epochs
+event_id, tmin, tmax = [1, 2], -0.2, 0.5
+
+# Setup for reading the raw data
+raw = mne.io.read_raw_fif(raw_fname, preload=True)
+raw.info['bads'] = ['MEG 2443', 'EEG 053']  # 2 bads channels
+events = mne.read_events(event_fname)
+
+# Set up pick list: gradiometers and magnetometers, excluding bad channels
+picks = mne.pick_types(raw.info, meg=True, eeg=False, stim=True, eog=True,
+                       exclude='bads')
+
+# Pick the channels of interest
+raw.pick_channels([raw.ch_names[pick] for pick in picks])
+
+# Re-normalize our empty-room projectors, so they are fine after subselection
+raw.info.normalize_proj()
+# Read epochs
+proj = False  # already applied
+epochs = mne.Epochs(raw, events, event_id, tmin, tmax,
+                    baseline=(None, 0), preload=True, proj=proj,
+                    reject=dict(grad=4000e-13, mag=4e-12, eog=150e-6))
+evoked = epochs.average()
+
+# Read regularized noise covariance and compute regularized data covariance
+noise_cov = mne.compute_covariance(epochs, tmin=tmin, tmax=0, method='shrunk')
+data_cov = mne.compute_covariance(epochs, tmin=0.04, tmax=0.15,
+                                  method='shrunk')
+# Read forward model
+forward = mne.read_forward_solution(fname_fwd)
+
+# Compute weights of free orientation (vector) beamformer with weight
```

Comment:
add a `#################################` (80 char) line before this so sphinx-gallery renders it as a separate section / narrative?